app/views/posts/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>New Post</h1>
+  <button><%=link_to 'Home', users_url, class: 'back-button' %></button>
+
+<%= form_with model: post, url: user_posts_path, method: :post, local: true, scope: :post do |f| %>
+
+  <div>
+    <%= f.label :title %><br>
+    <%= f.text_field :title %>
+  </div>
+
+  <div>
+    <%= f.label :text %><br>
+    <%= f.text_field :text %>
+  </div>
+
+  <%= f.submit "Create Post" %>
+
+<% end %>
```

Comment:
There is a form in the views/posts/new.html.erb file which accepts input for creating posts 🤓